ppr-ui/src/views/LengthTrust.vue

```diff
@@ -16,23 +16,16 @@
             </v-row>
             <stepper class="mt-4" />
             <v-row no-gutters class="pt-10">
-              <v-col cols="auto" class="sub-header" v-if="registrationType !== registrationTypeRL">
+              <v-col cols="auto" class="sub-header" v-if="registrationType !== registrationTypes['REPAIRERS_LIEN']">
```

Comment:
can you use the enum here?

Reply:
I'm certainly becoming less good with the options API... I just reverted back to the way Doug had it to keep it simple.